README.md

```diff
@@ -754,6 +754,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 
 * Argument parsing
   * [clap-rs](https://github.com/clap-rs/clap) [[clap](https://crates.io/crates/clap)] — A simple to use, full featured command-line argument parser [<img src="https://api.travis-ci.org/clap-rs/clap.svg?branch=master">](https://travis-ci.org/clap-rs/clap)
+  * [owez/climake](https://gitlab.com/Owez/climake) [[climake(https://crates.io/crates/climake)]] - A simple, dependency-free cli framework
```

Comment:
The markdown around "climake" is invalid. There should be a `]` after it and only one closing `]`

Reply:
Whoops, thanks!